### DISCUSSION TITLE
About maskmap and gauges

### DISCUSSION THREAD

**Discussion by weixiaoxun** [iiasa/CWatM#127] [Q&A]
Hello, Professor! I have a question regarding the coordinate settings for the MaskMap and gauges. The region I want to simulate is Northern China (Figure 1), which includes the outlet of the Yellow River. So I changed the coordinates to the location of the Yellow River shown in the figure. However, the simulated area only covers the Yellow River basin. How can I simulate the specific region of Northern China?
![{93453516-1B3D-4db9-A85B-27FCDCC4B418}](https://github.com/iiasa/CWatM/assets/98199745/f763bbe7-5c81-4afd-92fd-45842f2601d3)
![{709B67CD-2C20-4650-8CC6-E2C62371C08E}](https://github.com/iiasa/CWatM/assets/98199745/4433c5f4-da03-4c0c-b67e-e3796aa5b86b)

**Comment by weixiaoxun** [comment]
btw,I uploaded the first TIFF file to the folder, and set the MaskMap as follows: #MaskMap = $(FILE_PATHS:PathRoot)/areamaps/northchina30min.tif. However, I keep getting an error when it comes to the gauges. The error message is:

========================== CWATM ERROR =============================
Error 101: Gauges in settings file: D:\Users\Linze in D:\Users\Linze Wei\Documents\GitHub\CWatM-Earth-30min/routing/ldd.nc is not a coordinate.
Could it be that I have made a mistake in filling out the gauges? Thank you, Professor.

**Comment by weixiaoxun** [comment]
MaskMap, I followed your YouTube video to clip using ArcMap. Then, I clipped the ldd in ArcMap as well. I uploaded the paths for the two TIFF files, but I am still getting the Error 101.

**Comment by dof1985** [comment]
Hi @weixiaoxun 

Please provide the following values from your setting file:

[MASK_OUTLET]
MaskMap = $(FILE_PATHS:PathMaps)/areamaps/maskmap_adapted.nc

Gauges = 33.208 0.542 31.79 3.54 32.21 2.21

GaugesLocal = True

**Reply by weixiaoxun** [reply]
[](url)Thank you very much for your response, Professor! I immediately changed the path of the gauges to the coordinates you provided. However, when I run it, I still get error 101. Could it be a problem with my maskmap? I cropped the global.map provided in areamaps according to my region, but it still gives an error. Professor, could you please explain the meaning of the six numbers you provided? I would like to know how to fill in these numbers if I switch to another maskmap. Thank you for your answer, Professor. Thank you!](![QQ截图20240521170046](https://github.com/iiasa/CWatM/assets/98199745/ab31cf50-699d-4131-98f3-beb426a1d704))))))

**Reply by dof1985** [reply]
Hi @weixiaoxun 

I was probably not clear enough. Can you please send a screenshot with your current settings focusing on the [MASK_OUTLET] section, and the full error message?

**Reply by weixiaoxun** [reply]
Thank you for dof1985's response. Here is the information about [MASK_OUTLET]. Please help me check the following. Thank you so much!!!!
![%7BC4CE9EB3-90CB-4780-8EBE-1A0F64C15910%7D](https://github.com/iiasa/CWatM/assets/98199745/0d097b24-83eb-465d-9ae3-737b14817ca5)
![{FC11781C-F437-4fc2-BA67-2E51429C68AD}](https://github.com/iiasa/CWatM/assets/98199745/aecfbbca-f60f-43d5-b238-c73d03a9420c)

**Reply by dof1985** [reply]
Please replace Gauges with a xy pair that is in your mask map (those that you have put in are taken from lake Victoria area.

**Reply by weixiaoxun** [reply]
Thank you for your response. I replaced the gauges with a TIFF file, and it seems to be running now. However, I encountered a new error. I have set the StepStart as 01/01/1991, SpinUp as None, and StepEnd as 31/12/2010. But the simulation is ending prematurely as soon as it reaches 1/1/2010. Can you please assist me with this? Thank you.
![{CCB18CAF-5EBB-4bdd-89AF-EB94906D65F4}](https://github.com/iiasa/CWatM/assets/98199745/96336665-12f6-4f3e-861c-70707b58f434)
![{92E7C580-FBA9-466e-BEE8-78648BA52A01}](https://github.com/iiasa/CWatM/assets/98199745/d7f9a920-e5b5-45fd-aa33-9be4bf8cc76b)
